src/python/pants/backend/jvm/tasks/jvmdoc_gen.py

```diff
@@ -185,6 +189,17 @@ def _generate_individual(self, targets, create_jvmdoc_command):
           pool.terminate()
           self.context.log.debug("End multiprocessing section")
 
+  def _doc_classpath(self, lang_predicate, targets):
+    """Classpath including jar library targets and targets matching the language predicate."""
```

Comment:
Will excluding these libraries cause broken links in cases where they reference one another?